### DISCUSSION TITLE
Zaczynają blokować foliarskie strony

### DISCUSSION THREAD

**Discussion by xMaap** [FiltersHeroes/KAD#2101] [Ogólne]
ABW blokuje strony przez DNS. Znajduje się tam jedna ze stron która była w KAD https://github.com/FiltersHeroes/KAD/issues/1326 :smile_cat: 
`https://zaufanatrzeciastrona.pl/post/dlaczego-nie-dziala-mi-strona-czyli-jak-abw-walczy-z-kremlowska-propaganda/`

**Comment by hawkeye116477** [comment]
Czyli my zrezygnowaliśmy z tego tematu, a oni trochę jakby zrobili forka.

O jednej ze stron była też już dyskusja => https://github.com/FiltersHeroes/KAD/discussions/2070, a tu tymczasem najwyraźniej dodali kolejne.
